wolox-books/src/app/app-routing.module.ts

```diff
@@ -1,8 +1,24 @@
 import { NgModule } from '@angular/core';
 import { Routes, RouterModule } from '@angular/router';
+import { AppComponent } from './app.component';
+import { RegisterComponent } from  './screens/register/register.component';
+import { LoginComponent } from './screens/login/login.component'
 
 
-const routes: Routes = [];
+const routes: Routes = [
+  {
+    path: '', 
+    component: AppComponent
```

Comment:
You don't need to put this, use path: '', redirectTo: 'sign-up', pathmach: 'full'

Reply:
if you use this, when you go to the empty route you wont see anything